setup-guide/2a-setup-a-tracker/javascript-tracker/Integrating-Javascript-tags-with-enhanced-ecommerce.md

```diff
@@ -54,33 +54,38 @@ Whenever ecommerce data is pushed to the data layer, the tag will fire again. It
 
 The example script below will be used as the basis for your tag. There are some changes you should make to this script. In the example "SNOWPLOW_NAME_HERE" is used as the name of the Snowplow function. This is the global function used to make API calls to the Snowplow JavaScript Tracker. You should change this string to something unique so that if there is another Snowplow user on the page the namespaces will not collide. Similarly, you should change "MY_COOKIE_NAME" to a unique value. You should change "MY_COLLECTOR" to the URL of your Snowplow collector (minus the http/https scheme), for example "mycloudfrontsubdomain.cloudfront.net".
 
-You can also customize the part of the tag between the comments containing "!!!". The example below creates a tracker instance, sets page pings to fire every 10 seconds, and sends a page view event. See the [[JavaScript Tracker]] page for more information on other tracking methods.
+You can also customize the part of the tag between the comments containing "!!!". The example below creates a tracker instance, sets page pings to fire every 30 seconds, and sends a page view event. See the [[JavaScript Tracker]] page for more information on other tracking methods.
```

Comment:
Best to leave at 10, the setup tag has been changed to 10 to match the web model